src/ConvertESOMTR/mainModule.f90

```diff
@@ -430,6 +430,7 @@ Subroutine readESOdictionary()
 !'check to make sure the entire file wasn't treated as the data dictionary
 If (lineOfESO /= 'End of Data Dictionary') Then
```

Comment:
Yes I can. This looks like the correct fix so I would just run the defect file to confirm.